src/archetypes/java-web-app/openshift/deploy.yaml

```diff
@@ -0,0 +1,251 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: irs
+  annotations:
+    description: Deployment template for a java app.
+    tags: node
+    iconClass: icon-node
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}${SUFFIX}-tomcat
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: ${NAME}${SUFFIX}-tomcat
+  rules:
+  - apiGroups: [""] # "" indicates the core API group
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    creationTimestamp: null
+    name: ${NAME}${SUFFIX}-tomcat
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: ${NAME}${SUFFIX}-tomcat
+  subjects:
+  - kind: ServiceAccount
+    name: ${NAME}${SUFFIX}-tomcat
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    annotations: null
+    creationTimestamp: null
+    labels:
+      shared: "true"
+    name: ${NAME}
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: ${NAME}${SUFFIX}-log-pvc-vol
+  spec:
+    accessModes:
+      - ReadWriteMany
+    resources:
+      requests:
+        storage: ${LOG_PVC_SIZE}
+    storageClassName: netapp-file-standard
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: ${NAME}${SUFFIX}-app-data-vol
+  spec:
+    accessModes:
+      - ReadWriteMany
+    resources:
+      requests:
+        storage: ${APPDATA_PVC_SIZE}
+    storageClassName: netapp-file-standard
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: "${NAME}${SUFFIX}-webade-jar"
```

Comment:
Secret/"${NAME}${SUFFIX}-webade-jar" is not being used for deployment. needs to be removed from the deploy.yml.